carto/kuvizs.py

```diff
@@ -42,6 +43,28 @@ class Meta:
         collection_endpoint = API_ENDPOINT.format(api_version=API_VERSION)
         name_field = "id"
 
+    def update(self, html, name, password=None):
+        self.data = base64.b64encode(html.encode()).decode('ascii')
+        self.name = name
+
+        if password:
+            self.privacy = PRIVACY_PASSWORD
+            self.password = password
+        elif self.password:
```

Comment:
🤔 What does this mean?

Reply:
The user is changing the privacy from PASSWORD to PUBLIC. Initially, the Kuviz has a password, but now the user is removing it.

As I'm writing this, I realize that it does not make sense, because it should be explicit

Reply:
I am doubting

Reply:
I am going to change it to make all of them required. The behavior about password won't change, but it will be clear

Reply:
👍 

That logic is not easy to understand, what about adding a comment about what's happening?

Reply:
:+1:

Reply:
```
        if password:
            self.privacy = PRIVACY_PASSWORD
            self.password = password
        else:
            self.privacy = PRIVACY_PUBLIC
            self.password = None
```

Reply:
Thank you for insist :)

Reply:
One question I have about this is that it seems really easy to remove a password by accident. Please correct me if I'm wrong, because I'm not confident I'm seeing the whole picture here.

If a user wants to remove a password, they have to manually pass in `None` for the positional argument for `password`?

Reply:
Yes

Reply:
BTW, the rest of the picture is here https://github.com/CartoDB/cartoframes/pull/745